src/framework/uicomponents/view/popupwindow/popupwindow_qquickview.cpp

```diff
@@ -52,6 +52,7 @@ void PopupWindow_QQuickView::init(QQmlEngine* engine, bool isDialogMode, bool is
 
     m_view->setObjectName("PopupWindow_QQuickView");
     m_view->setResizeMode(QQuickView::SizeRootObjectToView);
+    m_view->setTitle("PopupWindow");
```

Comment:
Should there be a comment here, saying that this should be synced with winwindowscontroller?